### ISSUE TITLE
Maybe

### ISSUE THREAD

**Issue by MIGHTYCLIMBER** [Garmundo/Extended-unit-arsenal#6]
Add exclusive sandbox unit factory?

**Comment by Garmundo** [comment]
Yep will do. Some boss units I'm planning on making will be exclusively enemies in campaign. But I'll add a factory for them in sandbox. The supernova is the first example. I will add in the smaller units as factories tho like the aries and maybe the apparition
